### DISCUSSION TITLE
Fediverse, activitypub in joomla's future ?

### DISCUSSION THREAD

**Discussion by HDInfautre** [joomla/joomla-cms#41002] [2. Cold Features]
Hi
It seems that this sharing is promising and already used in other cms. See article https://magazine.joomla.org/all-issues/february-2023/turning-the-joomla-website-into-an-activitypub-server
I haven't seen any requests for this. 
Could joomla develop or promote these new features?

**Comment by wilsonge** [comment]
There's nothing on our short term roadmap for this - especially because as you mention there's not been that much demand for it. And mastodon usage has fallen back down a fair bit this year.

Futhermore we are aware of extensions in the Joomla ecosystem already doing this - such as https://github.com/nikosdion/fediverse so there's no major pressure on us as core to add such functionality at this moment in time.

Of course this doesn't rule out us doing anything in the future either :) Just where things stand at the moment

**Comment by HDInfautre** [comment]
> Of course this doesn't rule out us doing anything in the future either :) Just where things stand at the moment

Ok I understand I look at sharing. Note, a day later in native? Because it's not just related to Mastodon. 
This allows more generally to share in a transversal way of the content joomla :-)

**Comment by alikon** [comment]
i'll convert this "issue" to discussion
